doc/DataPlaneCodeGeneration/AzureSDKCodeGeneration_DataPlane_Quickstart.md

```diff
@@ -237,7 +222,8 @@ Adding convenience APIs is not required for Azure SDK data plane generated libra
 
 You can add convienice APIs by adding a customization layer on top of the generated code.  Please see the [autorest.csharp README](https://github.com/Azure/autorest.csharp#setup) for the details of adding the customization layer.  This is the preferred method for adding convenience APIs to your generated client.
 
-If other modifications are needed to the generated API, you can consider making them directly to the Open API specification, which will have the benefit of making the changes to the library in all languages you generate the library in.  As a last resort, you can add modifications with swagger transforms in the `autorest.md` file.  [AnomalyDetector autorest.md](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/anomalydetector/Azure.AI.AnomalyDetector/src/autorest.md) shows and example of how this can be accomplished.
+
+If other modifications are needed to the generated API, you can consider making them directly to the Open API specification, which will have the benefit of making the changes to the library in all languages you generate the library in.  As a last resort, you can add modifications with swagger transforms in the `autorest.md` file.  [DeviceUpdate autorest.md](https://github.com/Azure/azure-sdk-for-net/blob/3391ae91891f2b092c362a5e0696990c0f028dbc/sdk/deviceupdate/Azure.IoT.DeviceUpdate/src/autorest.md) shows and example of how this can be accomplished. Details for various transforms can be found in [Customizing the generated code](https://github.com/Azure/autorest.csharp#customizing-the-generated-code).
```

Comment:
Same above, it would be good to use the standalone sample file instead of refering a specific service.  If there are any design change in the future, we could just update the sample file.